### ISSUE TITLE
Is it matter if I put a test adapter to generate fastqc/multiqc/demux with --untilQ2import 

### ISSUE THREAD

**Issue by xkcococo** [nf-core/ampliseq#195]
Hi,

My codes is:
nextflow run nf-core/ampliseq -profile singularity --input seq/ --FW_primer GTGYCAGCMGCCGCGGTAA --RV_primer GGACTACNVGGGTWTCTAAT --untilQ2import --r 1.1.3 

Because I dont know the adapter for my seq data, and the value for --FW_primer GTGYCAGCMGCCGCGGTAA --RV_primer GGACTACNVGGGTWTCTAAT is fake because these two parameters are required. 
What I want to generate is fastqc/multiqc/demux to see the quality score plots for my samples. Is it matter if I put a fake adapter value here? For example, is it matter if it will effect the plots in fastqc?

Thanks in advance.
Ke

**Comment by erikrikarddaniel** [comment]
Why don't you just run fastqc and multiqc on their own?

**Comment by xkcococo** [comment]
> Why don't you just run fastqc and multiqc on their own?

Hi, thanks for helping.

I tried to use --untilQ2import, but I got an error when generating multiqc

Caused by:
  Process `multiqc` input file name collision -- There are multiple input files for each of the following file names: cutadapt/logs/cutadapt_log_1.txt, cutadapt/logs/cutadapt_log_2.txt

What's that mean and how could I adjust it?

Thanks in advance.
